### ISSUE TITLE
Emuhawk crashes when reloading N64 core with XSplit Game Source enabled.

### ISSUE THREAD

**Issue by danne2** [TASVideos/BizHawk#831]
Here's an interesting and extremely annoying bug for streamers.

If you're running XSplit and have the "Game Source" enabled, if you load an N64 game then take focus away from the Emuhawk window, then do anything that reloads the N64 core (manually reload the core, load a different N64 game, etc), the core will fail to load and you'll be greeted with a black screen.  If you then quit the program, you'll be greeted with a crash. If you do anything else to reload the core, you'll be greeted with a crash.

Disabling XSplit's "Game Source" fixes this issue but prevents optimal capture of the emulation output for streamers (as game capture is generally better than window capture. Also, window capture doesn't work properly with OpenGL mode)

Video demonstrating the bug:
https://streamable.com/j7vzd

Steps to reproduce:
Open XSplit (with Game Source enabled in the settings, on by default)
Open Emuhawk
Load N64 Rom
Switch focus from emuhawk to any other window and switch back
Reload the N64 Core in some way (manual reload, change games, etc)
You should be greeted with a black screen instead of the game you attempted loading, and if you attempt to load it again, it crashes.

This has been reproduced by a friend and I've reproduced it on multiple computers.

**Comment by zeromus** [comment]
game source is hacks. these hacks break the jabo plugin, which we don't have the source code to. If we had the source code, someone might could find out how xsplit is breaking jabo. But it's XSplit's problem to fix.

**Comment by danne2** [comment]
Re-opening this because I've found this same crash can be replicated with stuff other than XSplit.  It also happens with both Process Explorer and Process Hacker. I'm guessing it's anything querying for applications in a specific way? XSplit only crashes if you have game source enabled (which queries all your active applications every 10 seconds looking for things it can hook)

**Comment by zeromus** [comment]
Why did I see an email about this which didnt show up in the thread

**Comment by danne2** [comment]
Apologies.  I removed the post as I discovered my replication steps were incorrect.

I had a weird quirk where it'd crash when reloading the N64 core which went away after a reboot and I haven't been able to replicate. 

I thought it was related to having Process Explorer open (as after I opened Process Explorer it started happening) so I posted a response asking for it to be re-opened as I thought other programs were causing a very similar crash, but I haven't been able to replicate it after rebooting my computer.

All I can say for certain is it was related to Glide64MK2 as before I rebooted, I loaded up a second copy with the default settings and it was running fine. Went back to the version I had set to use Glide64Mk2 and it started crashing again. 

I'm working on replicating it but as of now I haven't been able to. Due to the fact it persisted across restarts of the application and went away after reboot, either something external was interfering (which is why I thought it was process explorer) or something outside of the emulator experienced an issue (graphics driver?). I'm working to replicate it and I'll open a new issue if I can properly replicate it.
